src/com/ableton/VirtualEnv.groovy

```diff
@@ -76,7 +79,10 @@ class VirtualEnv implements Serializable {
    */
   static VirtualEnv create(Object script, String python) {
     VirtualEnv venv = new VirtualEnv(script, python)
-    venv.script.sh("virtualenv --python=${python} ${venv.destDir}")
+    venv.script.sh(
+      label: "Create virtualenv for Python version ${python}",
```

Comment:
`python` is usually something like `python3.7`, so `Create virtualenv for Python version python3.7` reads a bit weird. What about `Create virtualenv for ${python}`?